CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.2.0
+* CHANGE: Callbacks set via `setDataCallback`, `setShouldSendCallback` now receive any prior-set callback as the 2nd argument. See: https://github.com/getsentry/raven-js/pull/636
+* CHANGE: Raven.js no longer passes a 'message' interface for exceptions. See: https://github.com/getsentry/raven-js/pull/632
```

Comment:
Also, does this mean the server will reject the event? Or just complain that the message interface is missing?